<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QDialog" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela Inicial</string>
  </property>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giro de Gado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="financialButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Financeiro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cattleButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="userButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Usu√°rio</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
